pkg/pb/synthetic_monitoring/checks.proto

```diff
@@ -123,7 +123,8 @@ message RegisterProbeResult {
 // Probe represents a probe.
 message Probe {
   message Capabilities {
-    bool disableScriptedChecks = 1 [(gogoproto.jsontag) = "disableScriptedChecks"]; // probe should not run scripted checks
+    bool disableScriptedChecks = 1 [(gogoproto.jsontag) = "disableScriptedChecks", deprecated = true]; // probe should not run scripted checks
```

Comment:
I considered renaming the existing field since it isn't in use yet, but `buf` has a `NO FIELD DELETE` validation which gets in the way... Maybe for the best